src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,81 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class HomePage extends BasePage {
+
+            public HomePage( WebDriver givenDriver) {
+                super(givenDriver);
+//                driver = givenDriver;//we can remove these ones since we have initialized them in basepage(inheritance using extends)
+//                wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        }
+
+//        POM locators
+//        By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+//        By playlistInputField = By.cssSelector("input[name='name']");
+//        By userAvatarIcon = By.cssSelector("img.avatar");
+
+//        Page Factory locators
+        @FindBy(css = "input[name='name']")
+        WebElement playlistInputField;
+
+        @FindBy(css = "img.avatar")
+        WebElement userAvatarIcon;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```